llama_hub/genius/base.py

```diff
@@ -0,0 +1,148 @@
+"""Genius Reader."""
+from typing import List, Optional
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+import lyricsgenius
```

Comment:
please see my comment in main conversation thread. Can we move this into the __init__ of the class?